data/AL_production.csv

```diff
@@ -0,0 +1,34 @@
+country,production[ktons/a],Year,source
```

Comment:
What happens with countries not on the list, is there any fallback or do we expect an error then?

Reply:
So far there will be an error. A fallback would be zero with a message stating that.